### PR TITLE
Fix #3459

### PR DESCRIPTION
**Pull request by rana-nouman**
http://prom.projects4.me/app/project/add12ff55acb-1902-6b9c-30d5411f7729/issue/3459

### CODE REVIEW

File: app/templates/app/index.hbs [hunk 1]
```diff
@@ -3,10 +3,35 @@
...
+      {{#each this.widgets as |value|}}
+        <div class="col-md-6 col-sm-12 widgets connectedSortable ui-sortable">
+          {{component (concat 'widgets/' (dasherize value)) data=(get-prop this value)}}
+        </div>
+      {{/each}}
+
+    {{else}}
+      <div class="col-md-12 text-center">
+        <img class="img-responsive" src="/img/app/logo-256.png" alt="Projects4me">
+        <h1>Welcome to <b>Projects</b>4<b>Me</b></h1>
```
Comment (gollomer) [inline app/templates/app/index.hbs]: translation

File: app/templates/app/index.hbs [hunk 2]
```diff
@@ -3,10 +3,35 @@
...
+    {{#if (gt this.currentUser.user.projects.length 0) }}
+      {{#each this.widgets as |value|}}
+        <div class="col-md-6 col-sm-12 widgets connectedSortable ui-sortable">
+          {{component (concat 'widgets/' (dasherize value)) data=(get-prop this value)}}
+        </div>
+      {{/each}}
+
+    {{else}}
+      <div class="col-md-12 text-center">
+        <img class="img-responsive" src="/img/app/logo-256.png" alt="Projects4me">
```
Comment (gollomer) [inline app/templates/app/index.hbs]: translation

File: app/templates/app/index.hbs [hunk 3]
```diff
@@ -3,10 +3,35 @@
...
+        <h1>Welcome to <b>Projects</b>4<b>Me</b></h1>
+
+        <AccessControl @aclContext="App.Project.Create">
+          <:default>
+            <p>{{t 'views.app.createNewProjectDescription'}}</p>
+            <LinkTo @route="app.projects.create">
+              <AppUi::Button type="button" class="btn btn-primary"><i class="fa fa-plus"></i> Create Project</AppUi::Button>
```
Comment (gollomer) [inline app/templates/app/index.hbs]: same

**Review by gollomer — CHANGES_REQUESTED** [review]

File: app/templates/app/index.hbs (hunk 1 above)
Comment (rana-nouman) [reply app/templates/app/index.hbs]: updated

File: app/templates/app/index.hbs (hunk 2 above)
Comment (rana-nouman) [reply app/templates/app/index.hbs]: updated

File: app/templates/app/index.hbs (hunk 3 above)
Comment (rana-nouman) [reply app/templates/app/index.hbs]: updated

**Review by gollomer — APPROVED** [review]

**Pull request merged**